app/src/components/explorer/Pagination.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
when I load locally, it seems like the pagination buttons don't show up when the screen is widest

Reply:
Just checking, @stouffers is the pagination now fixed based on this comment?

Reply:
fixed in latest set of commits